tools/RAiDER/cli/raider.py

```diff
@@ -21,6 +21,8 @@
 from RAiDER.models.allowed import ALLOWED_MODELS
 from RAiDER.utilFcns import get_dt
 
+import traceback
```

Comment:
I forget to get rid of this but eventually it should be implemented: see https://github.com/insarlab/MintPy/discussions/1017